src/BitcoinHelpers.js

```diff
@@ -45,12 +58,25 @@ const BitcoinNetwork = {
  */
 
 /**
- * @typedef {Object} SPVProof
- * @extends {Proof}
- * @property {ParsedTransaction} parsedTransaction Parsed transaction with
+ * @typedef {object} SPVProofInfo
+ * @property {HexTransactionFields} parsedTransaction Parsed transaction with
```

Comment:
Is it worth renaming the property to `hexTransactionFields` since the `ParsedTransaction` type no longer exists? Also fine keeping as is.

Reply:
I wanted to avoid logic changes where possible, but I think we'll be able to refactor this confidently with VS Code tooling after all of this stuff lands.